src/content/getusermedia/getdisplaymedia/index.html

```diff
@@ -35,7 +35,15 @@ <h1><a href="//webrtc.github.io/samples/" title="WebRTC samples homepage">WebRTC
 
     <video id="gum-local" autoplay playsinline muted></video>
     <button id="startButton" disabled>Start</button>
-
+    <fieldset id="options" style="display:none">
+      <legend>Advanced options</legend>
+      <select id="displaySurface">
+        <option value="default" selected>Show default sharing options</option>
+        <option value="browser">Prefer to share a browser tab</option>
+        <option value="window">Prefer to share a window</option>
+        <option value="monitor">Prefer to share the entire screen</option>
```

Comment:
```suggestion
        <option value="monitor">Prefer to share an entire screen</option>
```
There can be multiple screens plugged.